src/intl/en.json

```diff
@@ -398,14 +397,10 @@
     "section-2": "How It Works",
     "sash": {
       "first": {
-        "title": "One Pager",
-        "button": "Download"
-      },
-      "second": {
         "title": "White Paper",
         "button": "Download"
       },
-      "third": {
+      "second": {
```

Comment:
due to this change, we should update the text ids on the `about`, otherwise the buttons and the text is mixed 
![image](https://github.com/kleros/website/assets/80388279/a600d4e6-9a22-43bd-9d28-734e3ba6a003)